Jellyfin.Api/Controllers/TestsController.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Linq;
+using Jellyfin.Api.Constants;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// Controller for testing.
+    /// </summary>
+    public class TestsController : BaseJellyfinApiController
```

Comment:
This looks like debugging code

Reply:
it's the controller for the test sub-system. Need a controller to perform tests.

Reply:
moved to the test unit